ftest/web-ui/webdriver/test/features/clipboard.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Clipboard
+
+  I can move and copy from clipboard
+
+  Background:
+    Given I have the following documents
+    | doctype    | title            | nature  | subjects                | coverage             | creator | path                            | collections      | tag    | file       |
+    | Workspace  | Src              | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain                 |                  |        |            |
+    | Workspace  | Dest             | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain                 |                  |        |            |
+    | File       | File1            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
+    | File       | File2            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
+    | File       | File3            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
+    | File       | File4            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
+    | File       | File5            | booklet | sciences/astronomy      | europe/Portugal      | BJones  | /default-domain/Src             |                  |        |            |
+    And I login as "Administrator"
+
+  Scenario: Move from Clipboard
+    When I browse to the document at path "/default-domain/Src"
+    And I select all child documents
+    Then I can see the selection toolbar
+    And I can add selection to clipboard
+    When I click the "clipboard" button
+    Then I can see the clipboard has "5" items
+    When I browse to the document at path "/default-domain/Dest"
+    Then I can see the document has "0" children
+    When I click the clipboard move action
+    Then I can see the document has "5" children
+    And I can see the clipboard has "0" items
+    When I browse to the document at path "/default-domain/Src"
+    Then I can see the document has "0" children
+
+  Scenario: Paste from Clipboard
+    When I browse to the document at path "/default-domain/Src"
+    And I select all child documents
+    Then I can see the selection toolbar
+    And I can add selection to clipboard
+    When I click the "clipboard" button
+    Then I can see the clipboard has "5" items
+    When I browse to the document at path "/default-domain/Dest"
+    Then I can see the document has "0" children
+    When I click the clipboard paste action
+    Then I can see the document has "5" children
+    And I can see the clipboard has "0" items
+    When I browse to the document at path "/default-domain/Src"
```

Comment:
I'd say "with path" would make more sense, being `path` a property of the document, and later on we could even use "with id", and so on.

Reply:
ok, a Canadian native speaker suggested me `at path` but `with path` sounds good to me :)